src/context.js

```diff
@@ -145,12 +145,13 @@ function EvolvContext(store) {
       context = remoteContext;
     }
 
+    const thisRef = this;
```

Comment:
Not wrong, but I don't think capturing `this` as another variable was necessary.

Reply:
It was for some reason. Inside the forEach `this` was undefined. Idk why, do you?